<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Janky-PC" name="Tests" tests="9" failures="1" timestamp="2020-05-17T21:52:09 IST" time="97.817" errors="0">
  <testcase name="step1" time="13.102" classname="Tests"/>
  <system-out/>
  <testcase name="step2" time="22.042" classname="Tests"/>
  <system-out/>
  <testcase name="step3" time="8.830" classname="Tests"/>
  <system-out/>
  <testcase name="step4" time="6.500" classname="Tests"/>
  <system-out/>
  <testcase name="step5" time="5.290" classname="Tests"/>
  <system-out/>
  <testcase name="step6" time="0.234" classname="Tests"/>
  <system-out/>
  <testcase name="step7" time="10.749" classname="Tests"/>
  <system-out/>
  <testcase name="step8" time="30.494" classname="Tests">
    <failure type="java.lang.AssertionError" message="Start Free Trial not available for this course. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Start Free Trial not available for this course. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at Tests.step8(Tests.java:120)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
    </failure>
  </testcase> <!-- step8 -->
  <system-out/>
  <testcase name="step9" time="0.576" classname="Tests"/>
  <system-out/>
</testsuite> <!-- Tests -->
